dpe/src/commands/certify_key.rs

```diff
@@ -55,11 +55,13 @@ impl<C: Crypto> CommandExecution<C> for CertifyKeyCmd {
                         &self.label,
                         b"ECC",
                     )
+                    .unwrap()
```

Comment:
I think instead, you should be able to do something like this (in all the cases in this PR):

```rust
        let priv_key = if self.uses_nd_derivation() {
            if let Some(cached) = dpe.contexts[idx].cached_priv_key.take() {
                Ok(cached)
            } else {
                C::derive_private_key(
                    algs,
                    &dpe.derive_cdi(idx, true)?,
                    &self.label,
                    b"ECC",
                ).map_err(|_| DpeErrorCode::InternalError)
            }
        } else {
            let cdi = dpe.derive_cdi(idx, false)?;
            C::derive_private_key(algs, &cdi, &self.label, b"ECC")
                .map_err(|_| DpeErrorCode::InternalError)
        }?;
```